packages/icons/scripts/generateIconComponent.js

```diff
@@ -74,6 +71,29 @@ glob(globPath, options, function (error, filenames) {
         1
       );
 
+      // Some icon svg contain fill attributes which will break CSS var styling
+      htmlparser2.DomUtils.findAll(
+        (node) => {
+          if (node.attribs.fill) {
+            delete node.attribs.fill;
+          }
+        },
+        document.children,
+        true
+      );
+
+      // This strips fill-rule and clip-rule from the paths
```

Comment:
Removing these rules may result different render, would be good to find out which icons are having these and whether they're rendering the same or not

Reply:
I think SVGO or one of the other tools are combining `path`, e.g. `zip-solid.svg` has 4 `path` but in React, only 1. So removing them could cause render being different, e.g. warning solid is showing a solid triangle for me on Storybook preview.